src/index.js

```diff
@@ -61,6 +62,12 @@ const isWindows = 'navigator' in global && /Win/i.test(navigator.platform);
 const isMacLike =
   'navigator' in global && /(Mac|iPhone|iPod|iPad)/i.test(navigator.platform);
 
+function uid() {
```

Comment:
@kitten I used an uid because I wanted to prevent collision with user's CSS. But forgot about this! Good catch. I'll change it to something like `.npm__react-simple-editor`. Thank you!